app/styles/app.scss

```diff
@@ -323,10 +333,83 @@ input.light {
 
  }
 
+ .employee-button {
+   width: 119px;
+   height: 30px;
+   background-color: #fff;
+   font-weight: 400;
+   font-size: 14px;
+   line-height: 18px;
+   margin:10px;
+   margin-right:30px;
+
+ }
+
+ .slide-container0 {
+   right:0;
+   top:0;
+   position: absolute;
+   width: 100%;
+   height:100%;
+   z-index: 10;
+   
+   opacity:1 ;
+   display: flex;
+   justify-content: space-around;
+   @include animation(slide-modal .1s 1);
+
+}
+
+.outerslide{
+   
+   background: #12344D;
+   opacity: 0.3;
+   flex-grow: 2;
+   
+}
+
+.slide2 {
+   flex-grow: 1;
+   background-color: #fff;  
+}
 
+.slideheader {
+   margin:10px;
+   margin-left: 15px;
+   justify-content: space-between;
+}
 
+.buttons {
+   display: flex;
+}
 
+.headertext {
+   padding: 10px;
+}
 
+.button1 {
```

Comment:
.buttons, .button, .side are very generic class names. So, if we need to define module specific css properties, it's better to define with parent class hierarchy.

And, we can avoid names like button1, slide2. Use names like add-btn, save-btn, add-employee-slide etc..

Reply:
Updated.